hadoop-master1/Dockerfile

```diff
@@ -93,6 +99,9 @@ EXPOSE 9083 10000
 # SPARK ports
 EXPOSE 18080
 
+# FLINK ports
+EXPOSE 8081
```

Comment:
which service's port? for flink app web ui, should be accessible from yarn rm

Reply:
Yes, `8081` is Flink web UI's port.

> for flink app web ui, should be accessible from yarn rm

Let me remove and test it.